pkg/Microsoft.NETCore.Platforms/runtime.json

```diff
@@ -922,18 +915,11 @@
             "#import": [ "fedora-corert", "linux-x64-corert", "fedora-x64" ]
         },
 
-        "fedora.23-corert": {
-            "#import": [ "fedora-corert", "fedora.23" ]
-        },
-        "fedora.23-x64-corert": {
-            "#import": [ "fedora.23-corert", "fedora-x64-corert", "fedora.23-x64" ]
-        },
-
         "fedora.24-corert": {
```

Comment:
Is there a general policy on removing items from the RID graph? I have seen lots of additions to the RID graph, but never a removal.